Tests/Tests/CDTReplicationTests.m

```diff
@@ -33,6 +64,35 @@ @interface CDTReplicationTests : CloudantSyncTests
 
 @implementation CDTReplicationTests
 
+- (void)testFiltersWithChangesFeed
+{
+    NSError *error;
+    NSString *remoteUrl = @"https://example.com";
```

Comment:
Some comments here would be helpful - what we discussed about how this has to be a real URL because of the reachability tests, but we don't care that it there's not a real replicator on the end of this